src/fields/multi_field.hh

```diff
@@ -91,6 +91,12 @@ public:
     MultiField(const MultiField &other);
 
     /**
+     * Assignment operator. Same properties as copy constructor, but class member name_ is not copied.
+     */
```

Comment:
Note about preservation of field_names after set_component is called fro target multifield.